scipy/stats/_stats_py.py

```diff
@@ -9703,14 +9710,14 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
                 u\\
                 v\\
             \end{bmatrix}
-
+    The :math:`\text{vec}()` function denotes the Vectorization fucntion.
```

Comment:
```suggestion
    The :math:`\text{vec}()` function denotes the Vectorization function.
```
Although I've been doing this for a while, and I haven't heard of something referred to as a "Vectorization function".